x-pack/plugins/licensing/public/mocks.ts

```diff
@@ -18,7 +18,19 @@ const createSetupMock = () => {
   return mock;
 };
 
+const createStartMock = (): LicensingPluginStart => {
```

Comment:
Even if the impact is minor for that specific contract, returning the 'real' `LicensingPluginStart` type will hide the mock capabilities. (`createStartMock.refresh.mockImplementation` will raise a TS error)

Either remove the return type to let TS infer it (as it's done for the `setup`mock), or return `jest.Mocked<LicensingPluginStart>`